parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -1,7 +1,10 @@
-echo $(date),$(hostname) > /var/log/azure/cluster-provision-cse-output.log;
-cloud-init status --wait  > /dev/null 2>&1;
-[ $? -ne 0 ] && echo 'cloud-init failed' >> /var/log/azure/cluster-provision-cse-output.log && exit 1;
-echo "cloud-init succeeded" >> /var/log/azure/cluster-provision-cse-output.log;
+PROVISION_OUTPUT="/var/log/azure/cluster-provision-cse-output.log";
+echo $(date),$(hostname) > ${PROVISION_OUTPUT};
+{{if ShouldEnableCustomData}}
+cloud-init status --wait > /dev/null 2>&1;
```

Comment:
could we also add a test to baker_test.go to disable custom data and ensure that `cloud-init status --wait > /dev/null 2>&1` isn't present within cse?

Reply:
done :)